flight/Libraries/math/pid.c

```diff
@@ -199,6 +230,67 @@ void pid_configure(struct pid *pid, float p, float i, float d, float iLim)
 }
 
 /**
+ * Assigns a pid_deadband struct to the PID controller.
+ * @param[out] pid The PID structure to configure
+ * @param[in] deadband The external deadband structure to assign
+ */
+void pid_assign_deadband(struct pid *pid, struct pid_deadband *db)
+{
```

Comment:
I'm not sure about autotune as far as sane deadbands are concerned-- we should try with it first and see how it goes.

It might even be helpful, because it'll encourage undertuned behavior near the middle, and autotune likes undertuned conditions.